src/platform/packages/shared/kbn-monaco/src/esql/language.ts

```diff
@@ -26,6 +26,30 @@ const removeKeywordSuffix = (name: string) => {
   return name.endsWith('.keyword') ? name.slice(0, -8) : name;
 };
 
+// VSCODE DEFAULT FOUND HERE: https://github.com/microsoft/vscode/blob/1c931b181d6922ddc1eac2469117fba2c500da07/src/vs/editor/common/core/wordHelper.ts#L10
+export const ESQL_WORD_SEPARATORS = '`~!%^&*()-=+[{]}\\|;:",<>/';
+
+/**
+ * COPIED FROM VSCODE CODEBASE at https://github.com/microsoft/vscode/blob/1c931b181d6922ddc1eac2469117fba2c500da07/src/vs/editor/common/core/wordHelper.ts#L30-L48
+ *
+ * Create a word definition regular expression based on word separators.
+ * Optionally provide allowed separators that should be included in words.
+ *
+ * The default would look like this:
+ * /(-?\d*\.\d\w*)|([^\`\~\!\@\#\$\%\^\&\*\(\)\-\=\+\[\{\]\}\\\|\;\:\'\"\,\.\<\>\/\?\s]+)/g
+ */
+function createWordRegExp(allowInWords: string = ''): RegExp {
```

Comment:
This translation step allows us to share the `ESQL_WORD_SEPARATORS` between `wordPattern` in the language config and `wordSeparators` in the editor configuration. I also think it's easier to read a simple string than a big regex.